proposals/CSP.md

```diff
@@ -180,22 +180,30 @@ Proposed Changes:
 * This sub-proposal only affects WebAssembly.compileStreaming
   and WebAssembly.instatiateStreaming
   
-### Total Behavior
+### CSP Policy Application Summary
 
-The desired end state with both of these looks something like this:
+The checks performed for web assembly operations is summarized as follows:
 
-Operation | default | unsafe-eval | wasm-unsafe-eval | unsafe-eval + wasm-unsafe-val | w/ SRI ok | w/o SRI bad
+Operation | default | no unsafe-eval | with wasm-unsafe-eval | with unsafe-eval and wasm-unsafe-val 
```

Comment:
(I think there's a typo that seems to prevent the markdown from rendering as a table, at least in the GH preview.)

Reply:
Good catch, I didn't look at the preview. Missed updating the header row. PTAL.